cmd/accumulate/cmd/util.go

```diff
@@ -103,13 +103,13 @@ func prepareSigner(origin *url2.URL, args []string) ([]string, *transactions.Sig
 	}
 
 	bookRec := new(protocol.KeyBook)
-	if originRec.KeyBook == (types.Bytes32{}) {
+	if originRec.KeyBook == types.String("") {
```

Comment:
```suggestion
	if originRec.KeyBook == "" {
```

Go will automatically convert an untyped string literal to the correct type